src/check-for-spam.js

```diff
@@ -0,0 +1,41 @@
+// content refers to the contents of the blog post (article) sent by feedparser
+module.exports = function (content) {
+  // Minimum words set to 20, can be changed
+  const MINWORDS = 20;
+
+  // Functions
+  function removeTags(str) { // Removes HTML tags from a string, leaving only plaintext
```

Comment:
Use [text-parser.js](src/text-parser.js) instead

Reply:
text-parser.js being async was causing a lot of issues with both my code and the test scripts that I could not figure out how to work around, so I decided to leave them out, as I already had a working script.

Reply:
You should make your code expect to be async as well, and you'll have no issues.  Do you need help doing that?  If so, happy to guide you a bit.

Reply:
I tried making my function async as well as using await in the tests, but in tests it keeps returning undefined.

Reply:
Did you return a Promise?  An `async` function needs to return a `Promise`.  If you `await` something, it will use that `Promise`, but if not, you need to return a `Promise`, of if your code is purely synchronous, you can use `return Promise.resolve(yourReturnValue)`

Reply:
Alright I changed spam-checker to async and implemented text-parser